app/javascript/react/containers/ParkShowContainer.js

```diff
@@ -32,12 +47,31 @@ class ParkShowContainer extends Component {
   }
 
   render() {
+
+
     let reviewTiles = this.state.park.reviews.map (review => {
+      let thumbs_up = ''
+      let thumbs_down = ''
+      if(review.id === this.state.selectedThumbsUpId) {
+        thumbs_up = "thumbs_up"
+        thumbs_down = ""
```

Comment:
Because you set thumbs_down to "" immediately before this conditional, there's no need to set it here (same for line 60).